### ISSUE TITLE
Days and month stats don't appear on iphone browser

### ISSUE THREAD

**Issue by vezinaca** [vezinaca/banqorders#2]
I see all of the stats that I want to see on the book when I view app in browser.  But not when I view app on phone.  

For example: first book in list: I see 119 days (when viewing in browser) .  But see NaN days (in phone browser, both Safari AND Chrome)

**Comment by gregsadetsky** [comment]
you sometimes have `"last_checked": null,` in the `livres.json` file, which means that `new Date(null)` will not return a valid value

**Comment by gregsadetsky** [comment]
hmm... I'm less convinced now. Date(null) actually returns a valid date (it's a date in 1969... it's technically valid, although it's clearly not what you want)

**Comment by gregsadetsky** [comment]
right ok, the real problem is the following: you're storing the date in livres.json as a string, such as `2020-05-27 17:46:16` -- that's not a great strategy (to store dates as strings) because you can't really rely on the date parsing (unless you're using a library that does it well)

typically, storing dates is done by storing a timestamp, which is a number, which is the number of seconds since jan 1 1970... you can see some stuff around this here https://www.unixtimestamp.com/

so basically -- that's it. you have a string like "2020-05-27 17:46:16" and `new Date("2020-05-27 17:46:16")` will return an invalid date... and get calling .getDate() will return a NaN (not a number)
